.circleci/config.yml

```diff
@@ -7,76 +7,74 @@
 # To validate changes, use an online parser, eg.
 # http://yaml-online-parser.appspot.com/
 
-var_1: &docker_image circleci/node:10.16
+var_1: &docker_image circleci/node:12.9.1
+var_2: &docker-firefox-image
+  # TODO(devversion): Temporarily use a image that includes Firefox 62 because the
+  # ngcontainer image does include an old Firefox version that does not support headless.
```

Comment:
I think this reads a little better as `ngcontainer image includes an old Firefox`...

Reply:
I removed the whole comment since both images now match and it's no longer a workaround.